### PR TITLE
Fixed Edge versions to be withEdgeN versus withEdgeN_nnnn

### PR DESCRIPTION
**Pull request by nadvolod**
So rather than doing withEdge15_12345() we can now do withEdge15() and other Edge versions

### CODE REVIEW

File: java/src/main/java/com/saucelabs/simplesauce/IEVersion.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.saucelabs.simplesauce;
+
+public enum IEVersion {
+    ;
```
Comment (TheTestLynx) [inline java/src/main/java/com/saucelabs/simplesauce/IEVersion.java:4]: ;

File: java/src/main/java/com/saucelabs/simplesauce/SauceOptions.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        operatingSystem = "Windows 8";
+        return this;
+    }
+
+    public SauceOptions withWindows7() {
+        operatingSystem = "Windows 7";
+        return this;
+    }
+
+    public SauceOptions withEdge() {
```
Comment (TheTestLynx) [inline java/src/main/java/com/saucelabs/simplesauce/SauceOptions.java:94]: If this method is public maybe we should add the latest browserVersion too and change it if specified? Or have it private?

**Review by TheTestLynx — COMMENTED** [review]
Please check also the other comments about empty string and vista

File: java/src/main/java/com/saucelabs/simplesauce/IEVersion.java (hunk 1 above)
Comment (nadvolod) [reply java/src/main/java/com/saucelabs/simplesauce/IEVersion.java:4]: Yea, unless I missed something obvious, this is how the compiler forces me to do it in Java 👎 @joshmgrant does that look right to you?

File: java/src/main/java/com/saucelabs/simplesauce/SauceOptions.java (hunk 2 above)
Comment (nadvolod) [reply java/src/main/java/com/saucelabs/simplesauce/SauceOptions.java:94]: @TheTestLynx can you elaborate a bit more? Are you saying to be able to pass in a `edgeVersion` into this method?

**Pull request merged**